libs/ui/src/lib/activity-type/activity-type.component.stories.ts

```diff
@@ -0,0 +1,60 @@
+import { CommonModule } from '@angular/common';
+import { IonIcon } from '@ionic/angular/standalone';
+import { moduleMetadata } from '@storybook/angular';
+import type { Meta, StoryObj } from '@storybook/angular';
+
+import { GfActivityTypeComponent } from './activity-type.component';
+
+export default {
+  title: 'Activity Type',
+  component: GfActivityTypeComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [CommonModule, IonIcon]
+    })
+  ]
+} as Meta<GfActivityTypeComponent>;
+
+type Story = StoryObj<GfActivityTypeComponent>;
+
+export const Default: Story = {
+  args: {
+    activityType: 'BUY'
```

Comment:
Could you make the _Control_ a drop down list with all available options? Then, a single story will be enough.